homework/Vasili_Zasinets/ArrayServiceTask/src/main/java/com/tutrit/java/quickstart/homework/arrayservice/ArrayService.java

```diff
@@ -0,0 +1,54 @@
+package com.tutrit.java.quickstart.homework.arrayservice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Random;
+
+public class ArrayService {
+    Logger log = LoggerFactory.getLogger("ArrayService");
+    Random random = new Random();
+
+    public int[] create(int size) {
+        return new int[size];
+    }
+
+    public void fillRandomly(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            array[i] = random.nextInt(101);
+        }
+    }
+
+    public void printArray(int[] array) {
+        for (int j : array) {
+            log.info("array i = {} ", j);
+        }
+    }
+
+    public int sum(int[] array) {
+        int sumOfArray = 0;
+        for (int j : array) {
+            sumOfArray += j;
+        }
+        return sumOfArray;
+    }
+
+    public double avg(int[] array) {
+        if (verification(array) != 0) {
+            int sumOfArray = 0;
+            int average;
+            for (int j : array) {
+                sumOfArray += j;}
+            average = sumOfArray / (array.length);
```

Comment:
here will be mistake on type casting. Please, see comment in test and ask @lamaconda for expertise on this.

Reply:
ok